index_classes.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import os
+import textwrap
+import requests
+from utils import PollingProcess, events_to_process
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_issuers(height, class_id):
+    resp = requests.get(
+        f"{os.environ['REGEN_API']}/regen/ecocredit/v1/classes/{class_id}/issuers",
+        headers={"x-cosmos-block-height": str(height)},
+    )
+    resp.raise_for_status()
+    return resp.json()["issuers"]
+
+
+def _index_classes(pg_conn, _client, _chain_num):
```

Comment:
This function indexes credit class issuers, not credit classes themselves. We should make that clear.

```suggestion
def _class_issuers(pg_conn, _client, _chain_num):
```